services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -186,15 +187,20 @@ async def start_project_interactive_services(
 
 
 async def delete_project(app: web.Application, project_uuid: str, user_id: int) -> None:
-    await _delete_project_from_db(app, project_uuid, user_id)
+    """
 
-    async def _remove_services_and_data():
-        await remove_project_interactive_services(
-            user_id, project_uuid, app, notify_users=False
-        )
-        await storage_api.delete_data_folders_of_project(app, project_uuid, user_id)
+    raises ProjectDeleteError
+    """
+    try:
+        await _delete.mark_project_as_deleted(app, project_uuid)
```

Comment:
I understand that here you mark something to be deleted. I guess you do not wait for deletion.

Reply:
exactly. Following PR will also add an expiration date